Dockerfile

```diff
@@ -8,14 +10,16 @@ COPY package.json package-lock.json /usr/src/app/
 # Without the badge-maker package.json and CLI script in place, `npm ci` will fail.
 COPY badge-maker /usr/src/app/badge-maker/
 
-RUN npm install -g "npm@^10"
 # We need dev deps to build the front end. We don't need Cypress, though.
 RUN NODE_ENV=development CYPRESS_INSTALL_BINARY=0 npm ci
 
 COPY . /usr/src/app
-RUN npm run build
-RUN npm prune --omit=dev
-RUN npm cache clean --force
+
+RUN npm run build \
+    && npm prune --omit=dev --force \
+    && rm -rf node_modules/.cache \
+    && rm -rf frontend package-lock.json
```

Comment:
I think its probably fine to remove the source files once we are done with them, but just as a note: The lion's share of the gain here comes from removing the temp files the frontend build creates inside `frontend`, which can be done with `npm run docusaurus:clear` and `rm -rf frontend/categories`.